cmd/backup/storages/local.go

```diff
@@ -0,0 +1,118 @@
+package storages
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"time"
+
+	t "github.com/offen/docker-volume-backup/cmd/backup/types"
+	u "github.com/offen/docker-volume-backup/cmd/backup/utilities"
+	"github.com/sirupsen/logrus"
+)
+
+type LocalStorage struct {
+	*GenericStorage
+}
+
+// Specific init procedure for the local storage provider.
+func InitLocal(c *t.Config, l *logrus.Logger) *LocalStorage {
+	a := &GenericStorage{&LocalStorage{}, l, c}
+	r := &LocalStorage{a}
+	return r
+}
+
+// Specific copy function for the local storage provider.
+func (stg *LocalStorage) copy(file string) error {
+	_, name := path.Split(file)
+
+	if err := u.CopyFile(file, path.Join(stg.config.BackupArchive, name)); err != nil {
+		return fmt.Errorf("copyBackup: error copying file to local archive: %w", err)
```

Comment:
It would be great if the error strings map back to the actual function name, i.e. this should be `copy:` or even better `(*LocalStorage).copy:`

This applies to multiple places.